### ISSUE TITLE
"JxBrowser installation failed" (Fail to load "Flutter Inspector") in IntelliJ IDEA 2021.3

### ISSUE THREAD

**Issue by honood** [flutter/flutter-intellij#5876]
After upgrading the "Flutter" plugin to `63.0.5` in IntelliJ IDEA `2021.3`, my Flutter demo can run successfully. 
However, it showed me **"JxBrowser installation failed"** in "Flutter Inspector" panel, and no response when clicking the "Retry installation?" text button, related info in `idea.log` is: 

```
2021-12-02 21:45:49,617 [1096648]   INFO - ter.jxbrowser.JxBrowserManager - play_flutter: Retrying JxBrowser installation 
2021-12-02 21:45:49,617 [1096648]   INFO - ter.jxbrowser.JxBrowserManager - play_flutter: Installing JxBrowser 
2021-12-02 21:45:49,617 [1096648]   INFO - ter.jxbrowser.JxBrowserManager - play_flutter: JxBrowser platform files already exist, skipping download 
2021-12-02 21:45:49,617 [1096648]   INFO - ter.jxbrowser.JxBrowserManager - Loaded JxBrowser file successfully: ~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/213.5744.223/IntelliJ IDEA.app.plugins/flutter-intellij/jxbrowser/jxbrowser-mac-7.19.jar 
2021-12-02 21:45:49,618 [1096649]   INFO - ter.jxbrowser.JxBrowserManager - Loaded JxBrowser file successfully: ~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/213.5744.223/IntelliJ IDEA.app.plugins/flutter-intellij/jxbrowser/jxbrowser-7.19.jar 
2021-12-02 21:45:49,618 [1096649]   INFO - ter.jxbrowser.JxBrowserManager - Loaded JxBrowser file successfully: ~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/213.5744.223/IntelliJ IDEA.app.plugins/flutter-intellij/jxbrowser/jxbrowser-swing-7.19.jar 
2021-12-02 21:45:49,618 [1096649]   INFO - ter.jxbrowser.JxBrowserManager - Failed to find JxBrowser class 
```

By the way, I don't know why there are **two** item for "Android Studio" and "IntelliJ IDEA" respectively in the output of `flutter doctor -v` for a while.

## Version info

```
[✓] Flutter (Channel stable, 2.5.3, on macOS 12.0.1 21A559 darwin-x64, locale en-CN)
    • Flutter version 2.5.3 at ~/Dev/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (7 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at ~/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: ~/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7784292/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 13.1, Build version 13A1030d
    • CocoaPods version 1.11.2

[✓] Android Studio (version 2020.3)
    • Android Studio at ~/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7784292/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] Android Studio (version 2020.3)
    • Android Studio at ~/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/203.7784292/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[☠] IntelliJ IDEA Ultimate Edition (the doctor check crashed)
    ✗ Due to an error, the doctor check did not complete. If the error message below is not helpful, please let us know about this issue at
      https://github.com/flutter/flutter/issues.
    ✗ FormatException: Unexpected extension byte (at offset 5)
    • #0      _Utf8Decoder.convertSingle (dart:convert-patch/convert_patch.dart:1788:7)
      #1      Utf8Decoder.convert (dart:convert/utf.dart:318:42)
      #2      InputStream.readString (package:archive/src/util/input_stream.dart:199:30)
      #3      new ZipDirectory.read (package:archive/src/zip/zip_directory.dart:40:30)
      #4      ZipDecoder.decodeBuffer (package:archive/src/zip_decoder.dart:19:30)
      #5      ZipDecoder.decodeBytes (package:archive/src/zip_decoder.dart:14:12)
      #6      IntelliJPlugins._findPluginXml (package:flutter_tools/src/intellij/intellij.dart:130:44)
      #7      IntelliJPlugins._readPackageVersion (package:flutter_tools/src/intellij/intellij.dart:141:40)
      #8      IntelliJPlugins.validatePackage (package:flutter_tools/src/intellij/intellij.dart:63:35)
      #9      IntelliJValidator.validate (package:flutter_tools/src/intellij/intellij_validator.dart:103:15)
      #10     asyncGuard.<anonymous closure> (package:flutter_tools/src/base/async_guard.dart:111:32)
      #11     asyncGuard.<anonymous closure> (package:flutter_tools/src/base/async_guard.dart:109:18)
      #12     _rootRun (dart:async/zone.dart:1428:13)
      #13     _CustomZone.run (dart:async/zone.dart:1328:19)
      #14     _runZoned (dart:async/zone.dart:1863:10)
      #15     runZonedGuarded (dart:async/zone.dart:1851:12)
      #16     runZoned (dart:async/zone.dart:1782:12)
      #17     asyncGuard (package:flutter_tools/src/base/async_guard.dart:109:3)
      #18     Doctor.startValidatorTasks (package:flutter_tools/src/doctor.dart:200:9)
      #19     Doctor.diagnose (package:flutter_tools/src/doctor.dart:304:47)
      #20     DoctorCommand.runCommand (package:flutter_tools/src/commands/doctor.dart:50:47)
      #21     FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:1274:12)
      <asynchronous suspension>
      #22     FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1125:27)
      <asynchronous suspension>
      #23     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
      <asynchronous suspension>
      #24     CommandRunner.runCommand (package:args/command_runner.dart:209:13)
      <asynchronous suspension>
      #25     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:288:9)
      <asynchronous suspension>
      #26     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
      <asynchronous suspension>
      #27     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:236:5)
      <asynchronous suspension>
      #28     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
      <asynchronous suspension>
      #29     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
      <asynchronous suspension>
      #30     main (package:flutter_tools/executable.dart:92:3)
      <asynchronous suspension>


[✓] IntelliJ IDEA Ultimate Edition (version 2021.3)
    • IntelliJ at ~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/213.5744.223/IntelliJ IDEA.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.62.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.29.0

[✓] Connected device (2 available)
    • iPhone 13 (mobile) • DA1BE53F-5B7D-4CE7-9D5B-1B10C98AA3A6 • ios        • com.apple.CoreSimulator.SimRuntime.iOS-15-0 (simulator)
    • macOS (desktop)    • macos                                • darwin-x64 • macOS 12.0.1 21A559 darwin-x64

! Doctor found issues in 1 category.
```

**Comment by honood** [comment]
Any Flutter member could help to fix this issue? 
Not only “IntelliJ IDEA”, but also “Android Studio” cannot load “Flutter Inspector” successfully.

**Comment by honood** [comment]
It seems that the reason the doctor check crashed when reading the `Dart` plu

From the above crash backtrace and [flutter code](https://github.com/flutter/flutter/blob/2.5.3/packages/flutter_tools/lib/src/intellij/intellij_validator.dart#L103), it seems that the crash was triggered when invalidating `Dart` plugin.

![image](https://user-images.githubusercontent.com/665595/144656245-fbfbbef8-483e-4636-bcb3-d1c74a45b5d3.png)

**Comment by tallinn1960** [comment]
Flutter plugin 63.0.5 on 2021.3 does not work on M1 as well. Idea.log shows:
.
2021-12-05 12:51:29,700 [ 234232]   INFO - ter.jxbrowser.JxBrowserManager - Failed to find JxBrowser class

**Comment by honood** [comment]
The issue has been fixed after upgrading to Flutter plugin 63.1.4 in my dev env.

**Comment by tallinn1960** [comment]
I can confirm: issue fixed on M1 as well. And no workaround for loading the wrong chromium binary needed any longer.
